Entity/CategoryManager.php

```diff
@@ -136,6 +176,31 @@ public function getRootCategories($loadChildren = true)
     }
 
     /**
+     * @param bool|true $loadChildren
+     *
+     * @return array
+     */
+    public function getRootCategoriesSplitByContexts($loadChildren = true)
```

Comment:
Can you please add a `NEXT_MAJOR` comment to all new public methods, so we know that we should add them to the interface.